common/workunit/workunit.ipp

```diff
@@ -87,6 +86,20 @@ public:
     virtual bool matches(const IStatisticsFilter * filter) const;
 };
 
+class CLocalWUFieldUsageFile : public CInterface, implements IConstWUFieldUsageFile
+{
+    Owned<IPropertyTree> p;
+public:
+    IMPLEMENT_IINTERFACE;
+    CLocalWUFieldUsageFile(IPropertyTree& _p) { p.setown(&_p); }
+
+    virtual IStringVal & getName(IStringVal & ret) const { ret.set(p->queryProp("@name")); return ret; }
+    virtual IStringVal & getType(IStringVal & ret) const { ret.set(p->queryName()); return ret; }
+    virtual unsigned getNumFields() const { return p->getPropInt("@numFields"); }
+    virtual unsigned getNumFieldsUsed() const { return p->getPropInt("@numFieldsUsed"); }
+    virtual IPropertyTreeIterator * getFields() const { return p->getElements("field"); }
```

Comment:
This is okay for now (IPropertyTreeIterator, but I have a feeling we will end up converting it to an IConstWUFieldUsageFieldIterator before we are done